pkg/autoscaler/metrics/stats_scraper_test.go

```diff
@@ -91,6 +96,30 @@ func TestNewServiceScraperWithClientHappyCase(t *testing.T) {
 	}
 }
 
+func checkBaseStat(t *testing.T, got Stat) {
+	if got.PodName != scraperPodName {
+		t.Errorf("stat.PodName=%v, want %v", got.PodName, scraperPodName)
+	}
+	// (3.0 + 5.0 + 3.0) / 3.0 * 3 = 11
+	if got.AverageConcurrentRequests != 11.0 {
+		t.Errorf("stat.AverageConcurrentRequests=%v, want %v",
+			got.AverageConcurrentRequests, 11.0)
+	}
+	// ((5 + 7 + 5) / 3.0) * 3 = 17
+	if got.RequestCount != 17 {
+		t.Errorf("stat.RequestCount=%v, want %v", got.RequestCount, 15)
```

Comment:
```suggestion
		t.Errorf("stat.RequestCount=%v, want %v", got.RequestCount, 17)
```

Reply:
Hah that one I think was in the Original as well :)